### DISCUSSION TITLE
Change name of inverter model

### DISCUSSION THREAD

**Discussion by miralemR** [nathanmarlor/foxess_modbus#488] [Support] [answered]
Hello,

I tried to instal FoxEss modbus integration and I get this message:

Error connecting to your adapter or inverter. Ensure the adapter is properly configured and is correctly wired to your inverter (see the setup link above), then try again. Details: Inverter model not supported: 'STAR-H3-12.0-E'.

I can see a different name of inverter, but it is actually made from the FoxEss for another energy company in Slovenia. It si the same user manual, menu, shape, color... everithing but name. Instead of FoxEss sticker above the display there it says ngen ess.

Is there a way to bypass name STAR in inverter model?

**Comment by canton7** [comment, marked as answer]
I'll have to add support for it

**Reply by canton7** [reply]
See #489

**Reply by miralemR** [reply]
Thank you for your fast respond.

**Reply by canton7** [reply]
I'll hopefully be able to get to it over the weekend, but #480 is higher priority right now

**Reply by canton7** [reply]
Do you know if there's a STAR-H1 version?

**Reply by canton7** [reply]
STAR-H3 support added in v1.8.1.

**Reply by miralemR** [reply]
Yes. These are the versions... STAR S2/S3/S4/S5/S6/S7
